src/elm/Main.elm

```diff
@@ -1,4 +1,4 @@
-module Main exposing (main)
+module Main exposing (AuthError, Flags, Model, Msg, UserData, main)
```

Comment:
Uh, what? This should never be necessary. I don't understand these other types being added to `exposing` either. If `elm-review` requires these, it seems to be broken and should be fixed.

Reply:
Reglen som slår inn her tror jeg at man skal eksportere typer er på funksjoner som blir eksportert. åpen for at det ikke er nødvendig her og at reglen blir for dogmatisk.

Reply:
For the `Main` module this doesn't make sense at least. For other modules, I can agree with the types that are listed in the function definition, but not necessarily types contained within. For example, in `Page.Login`, `Step` is an internal implementation detail. You _never_ create the model manually outside of the module, that is done by the `init` function. Same with updating it, that is done by `update`. Thus `Step` should never be exposed.

Reply:
Removed rule now. Can at some point evaluate if we want to have it included but some modules ignored for instance. I'm not really a fan of to dogmatic rules of this kind, but here it is to just ensure compile validation and some code base practises you pointed out in a previous code review.

Reply:
Great, thanks! We can probably look into making a better rule for this that doesn't force you to expose types that shouldn't be exposed. If it doesn't compile, hopefully the CI step should take care of that (for now).